docs/data/data-grid/export/export.md

```diff
@@ -185,6 +185,10 @@ The demo below shows how to add a JSON export.
 
 {{"demo": "CustomExport.js", "bg": "inline", "defaultCodeOpen": false}}
 
+> ⚠️ Due to the fact that the Print export relies on the usage of an `iframe` to prepare and only print the data grid and not the entire page, there is a limitation around the usage of `X-Frame-Options`.
```

Comment:
The reason why we use iframe is not very interesting for readers. All they want to know is how to make it work

```suggestion
> ⚠️ Due to the fact that the Print export relies on the usage of an `iframe`, there is a limitation around the usage of `X-Frame-Options`.
```